packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -382,9 +397,15 @@ class _AppBarState extends State<AppBar> {
     final bool canPop = parentRoute?.canPop ?? false;
     final bool useCloseButton = parentRoute is PageRoute<dynamic> && parentRoute.fullscreenDialog;
 
-    IconThemeData appBarIconTheme = widget.iconTheme ?? themeData.primaryIconTheme;
-    TextStyle centerStyle = widget.textTheme?.title ?? themeData.primaryTextTheme.title;
-    TextStyle sideStyle = widget.textTheme?.body1 ?? themeData.primaryTextTheme.body1;
+    IconThemeData appBarIconTheme = widget.iconTheme
+        ?? appBarTheme.iconTheme
```

Comment:
2 spaces of indent here.
```
IconThemeData appBarIconTheme = widget.iconTheme
  ?? appBarTheme.iconTheme
  ?? themeData.primaryIconTheme;
 ```

Same below.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#indent-multi-line-argument-and-parameter-lists-by-2-characters

Reply:
Done